mySQL-example/MySQL.go

```diff
@@ -0,0 +1,183 @@
+package scripts
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+	//MySQL driver
+	_ "github.com/go-sql-driver/mysql"
+	//convert to strings
+	// import of application related stuff
+	"git.monkey-works.de/scripting/api"
+	"monkey-works.de/model"
+)
+
+//The endpoint of our web service we want to connect to
+//Change this to your endpoint
+const URL string = "tcp(127.0.0.1:3306)/schema" //tcp(yourHost_IP:Port)/Schema_name
+const USERNAME string = "user"
+const PASSWORD string = "1234"
+const QUERY string = "SELECT * FROM variables WHERE id = ?" //your SQL Statement, "?" is a placeholder
+const UPDATE_INTERVAL time.Duration = 2000
+
+var db *sql.DB //you database
+
+var id int
+
+var app model.Application //app stuff, we'll need it for the DataItems
+
+var status model.StringDataItem //Status Item to show errors and progress
+
+//handle errors and stop the database connection
+var ok bool
+var Stop bool
+
+/*
+ _________________________________
+|  id  |    name    |    value    |
+|______|____________|_____________|
+|  1   | DateItem1  |  HelloWorld |
+|______|____________|_____________|
+*/
+// this struct is what we need/get from our SQL Statement
+type SQLData struct {
+	id    int
+	name  string
+	value string
+}
+
+// @script
+func InitializeScripting(application model.Application) {
+	fmt.Println("Hello Scripting")
+	// get status item
+	status, ok = application.ClientDataModel().FindDataItemByName("status").(model.StringDataItem)
+	if !ok {
+		panic("DataItem \"status\" not found")
+	}
+	app = application
+	// create start and stop button
+	registerButton()
+	registerStopButton()
+}
+
+func CheckConnectionToDataBase() bool {
```

Comment:
Can you please add documentation about this method, what it does and so on. See https://blog.golang.org/godoc-documenting-go-code for recommendations on how to doc go code.